torchvision/models/segmentation/deeplabv3.py

```diff
@@ -3,6 +3,7 @@
 from torch.nn import functional as F
 
 from ._utils import _SimpleSegmentationModel
+from typing import List
```

Comment:
Can you do `from torch.jit.annotations import List` instead?

Reply:
We can - however now in annotations.py we're just importing from typing anyway  https://github.com/pytorch/pytorch/pull/27510/

Reply:
This hasn't been merged in PyTorch yet, and we will be cutting a new release of TorchVision in the next couple of weeks, so it should ideally make torchvision 0.5 (or 0.4.2) compatible with PyTorch 1.3

Reply:
Actually, this import is not used in the code?